vilebot/src/main/java/com/oldterns/vilebot/handlers/user/News.java

```diff
@@ -0,0 +1,202 @@
+/**
+ * Copyright (C) 2019 Newterns
```

Comment:
Newterns should be changed to Oldterns afaik

Reply:
B-B-But but I'm a newtern 0.o
Noted.